partitioned-table.md

```diff
@@ -582,11 +582,19 @@ For `HASH` partitioned tables, `COALESCE PARTITION` and `ADD PARTITION` are not
 
 For example, `ALTER TABLE partitioned_table EXCHANGE PARTITION p1 WITH TABLE non_partitioned_table` swaps the `non_partitioned_table` table in the `p1` partition with the `partitioned_table` table.
 
-Ensure that all rows that you are exchanging into the partition match the partition definition; otherwise, these rows will not be found and cause unexpected issues.
+Ensure that all rows that you are exchanging into the partition match the partition definition; otherwise, exchange will fail.
```

Comment:
```suggestion
Ensure that all rows that you are exchanging into the partition match the partition definition; otherwise, the exchange will fail.
```